src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -159,7 +159,7 @@ public enum FormatterTypeStyle
     }
     public partial interface IFieldInfo
     {
-        string[] FieldNames { get; set; }
+        string[]? FieldNames { get; set; }
         System.Type[] FieldTypes { get; set; }
```

Comment:
Didn't found any implementation of this interface, found a doc that FieldNames could return null, but nothing mentioned about FieldTypes nullability.